aapcs64-morello/aapcs64-morello.rst

```diff
@@ -674,26 +679,28 @@ The differences in language bindings used for AAPCS64 and AAPCS64-cap are descri
-    |                               | The NSAA is rounded up to the larger of 8 or the Natural Alignment of the argument's   |
+    |                               | The NMAA is rounded up to the larger of 8 or the Natural Alignment of the argument's   |
     |                               | type.                                                                                  |
     | C.15                          |                                                                                        |
     +-------------------------------+----------------------------------------------------------------------------------------+
     |                               | If the argument is a composite type then the argument is copied to memory at the       |
-    |                               | adjusted NSAA. The NSAA is incremented by the size of the argument. The argument has   |
+    |                               | adjusted NMAA. The NMAA is incremented by the size of the argument. The argument has   |
     | C.16                          | now been allocated.                                                                    |
     +-------------------------------+----------------------------------------------------------------------------------------+
     |                               | If the size of the argument is less than 8 bytes then the size of the argument is set  |
     |                               | to 8 bytes. The effect is as if the argument was copied to the least significant bits  |
     | C.17                          | of a 64-bit register and the remaining bits filled with unspecified values.            |
     +-------------------------------+----------------------------------------------------------------------------------------+
-    |                               | The argument is copied to memory at the adjusted NSAA.  The NSAA is incremented by the |
+    |                               | The argument is copied to memory at the adjusted NMAA.  The NMAA is incremented by the |
     |                               | size of the argument. The argument has now been allocated.                             |
     | C.18                          |                                                                                        |
     +-------------------------------+----------------------------------------------------------------------------------------+
 
 .. note::
+    In AAPCS64-cap all Anonymous arguments are memory passed.
 
-    In AAPCS64-cap if the callee is variadic and there are fewer than 4096 Anonymous arguments, the length of C9 divided by 16 is equal to the number of Anonymous arguments. The length of C9 divided by 16 is always greater or equal to the number of Anonymous arguments.
+.. note::
+    In AAPCS64-cap if the callee is variadic the number of Anonymous arguments is equal to (length(C)-offset(C)/16) where C is the adjusted value of C9 that has the address of the first Anonymous Argument.
```

Comment:
Should this be `(length(C)-offset(C))/16` instead?

Reply:
Yes, thanks for catching that.